contracts/src/challengeV2/libraries/EdgeChallengeManagerLib.sol

```diff
@@ -469,6 +469,53 @@ library EdgeChallengeManagerLib {
         return (hasRival(store, edgeId) && store.edges[edgeId].length() == 1);
     }
 
+    function updateAccuTimerCache(EdgeStore storage store, bytes32 edgeId, bytes32 claimingEdgeId, uint8 numBigStepLevel) internal returns (uint64) {
+        // calculate the time unrivaled without inheritance
+        // we don't use timeAccumulated here, since it would read the cache and double-count
+        uint64 accuTimer = timeUnrivaled(store, edgeId);
+        if (store.edges[edgeId].lowerChildId != bytes32(0)) {
+            uint64 lowerTimer = timeAccumulated(store, store.edges[edgeId].lowerChildId, numBigStepLevel);
+            uint64 upperTimer = timeAccumulated(store, store.edges[edgeId].upperChildId, numBigStepLevel);
+            if (upperTimer == type(uint64).max && lowerTimer == type(uint64).max) {
+                return type(uint64).max;
+            }
+            accuTimer += lowerTimer < upperTimer ? lowerTimer : upperTimer;
```

Comment:
if the edge is a root edge (layer zero), we need to use the max function for comparison here